cluster-autoscaler/core/utils/utils.go

```diff
@@ -42,7 +42,7 @@ import (
 const (
 	// ReschedulerTaintKey is the name of the taint created by rescheduler.
 	ReschedulerTaintKey = "CriticalAddonsOnly"
-	// IgnoreTaintPrefix any taint starting with it will be filtered out from autoscaler template node.
+	// IgnoreTaintPrefix any taint starting with it will be filtered out from autoscaler nodes.
```

Comment:
I think the change actually makes the comment harder to understand. Either expand it further or revert (old wording is actually not far from truth - marking node as unready make CA use template node to represent it).

Reply:
Will do !